### PR TITLE
LF: Add TO_TEXT_BIGNUMERIC

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
This is part of #8719.

CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto [hunk 1]
```diff
@@ -495,13 +495,13 @@ enum BuiltinFunction {
...
 
   ERROR = 25;
-  ANY_EXCEPTION_MESSAGE = 146;      // *Available in versions >= 1.dev*
-  MAKE_GENERAL_ERROR = 147;         // *Available in versions >= 1.dev*
-  GENERAL_ERROR_MESSAGE = 148;      // *Available in versions >= 1.dev*
-  MAKE_ARITHMETIC_ERROR = 149;      // *Available in versions >= 1.dev*
-  ARITHMETIC_ERROR_MESSAGE = 150;   // *Available in versions >= 1.dev*
-  MAKE_CONTRACT_ERROR = 151;        // *Available in versions >= 1.dev*
-  CONTRACT_ERROR_MESSAGE = 152;     // *Available in versions >= 1.dev*
+  ANY_EXCEPTION_MESSAGE = 147;      // *Available in versions >= 1.dev*
```
Comment (cocreature) [inline daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto:498]: Why change the existing numbers? I know it’s only 1.dev so this is fine but still doesn’t seem all that useful.

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto (hunk 1 above)
Comment (remyhaemmerle-da) [reply daml-lf/archive/src/main/protobuf/com/daml/daml_lf_dev/daml_lf_1.proto:498]: I just find that cleaner

**Review by sofiafaro-da — APPROVED** [review]
Thanks! LGTM!

**Comment by remyhaemmerle-da** [comment]
part of #8719

**Pull request merged**